### ISSUE TITLE
Add ntpsrv to PropertyTable 

### ISSUE THREAD

**Issue by erauer** [nerves-networking/vintage_net#440]
I would like to be able to retrieve dhcpc attributes specifically the `ntpsrv` property.

On my home network, I have an ntp server returned via dhcp . On a running raspberry pi 4 with nerves, the following properties are logged from `VintageNet.Interface.Udhcpc.renew/2`:

```
%{"BINDIR" => "/srv/erlang/erts-13.1/bin", "EMU" => "beam", "ERL_CRASH_DUMP" => "/root/crash.dump", "ERL_INETRC" => "/etc/erl_inetrc", "HEART_BEAT_TIMEOUT" => "30", "HOME" => "/root", "LANG" => "en_US.UTF-8", "LANGUAGE" => "en", "PATH" => "/srv/erlang/erts-13.1/bin:/srv/erlang/bin:/usr/sbin:/usr/bin:/sbin:/bin", "PROGNAME" => "erlexec", "RELEASE_ROOT" => "/srv/erlang", "RELEASE_SYS_CONFIG" => "/srv/erlang/releases/1.8.15-2/sys", "RELEASE_TMP" => "/tmp", "ROOTDIR" => "/srv/erlang", "SHELL" => "/bin/sh", "TERM" => "vt100", "dns" => ["192.168.1.149", "1.1.1.1", "9.9.9.9"], "domain" => "localdomain", "interface" => "eth0", "ip" => "192.168.1.245", "lease" => "86400", "mask" => "24", "ntpsrv" => "192.168.1.149", "opt53" => "05", "router" => ["192.168.1.1"], "serverid" => "192.168.1.1", "subnet" => "255.255.255.0"}
```

Some of  info from the dhcp response is logged in PropertyTable but not all of it:
```
iex(1)> PropertyTable.get_all(VintageNet)
[
  {["connection"], :internet},
  {["interface", "eth0", "type"], VintageNetEthernet},
  {["interface", "eth0", "connection"], :internet},
  {["interface", "eth0", "present"], true},
  {["interface", "lo", "lower_up"], true},
  {["available_interfaces"], ["eth0"]},
  {["interface", "eth0", "addresses"],
   [
     %{
       address: {65010, 23786, 29427, 17390, 58975, 511, 65138, 47810},
       family: :inet6,
       netmask: {65535, 65535, 65535, 65535, 0, 0, 0, 0},
       prefix_length: 64,
       scope: :universe
     },
     %{
       address: {9729, 1605, 49920, 11043, 58975, 511, 65138, 47810},
       family: :inet6,
       netmask: {65535, 65535, 65535, 65535, 0, 0, 0, 0},
       prefix_length: 64,
       scope: :universe
     },
     %{
       address: {192, 168, 1, 245},
       family: :inet,
       netmask: {255, 255, 255, 0},
       prefix_length: 24,
       scope: :universe
     },
     %{
       address: {65152, 0, 0, 0, 58975, 511, 65138, 47810},
       family: :inet6,
       netmask: {65535, 65535, 65535, 65535, 0, 0, 0, 0},
       prefix_length: 64,
       scope: :link
     }
   ]},
  {["interface", "wlan0", "hw_path"],
   "/devices/platform/soc/fe300000.mmcnr/mmc_host/mmc1/mmc1:0001/mmc1:0001:1"},
  {["interface", "lo", "mac_address"], "00:00:00:00:00:00"},
  {["interface", "eth0", "mac_address"], "e4:5f:01:72:ba:c2"},
  {["interface", "eth0", "lower_up"], true},
  {["interface", "lo", "hw_path"], "/devices/virtual"},
  {["interface", "wlan0", "present"], true},
  {["interface", "wlan0", "lower_up"], false},
  {["interface", "lo", "addresses"],
   [
     %{
       address: {0, 0, 0, 0, 0, 0, 0, 1},
       family: :inet6,
       netmask: {65535, 65535, 65535, 65535, 65535, 65535, 65535, 65535},
       prefix_length: 128,
       scope: :host
     },
     %{
       address: {127, 0, 0, 1},
       family: :inet,
       netmask: {255, 0, 0, 0},
       prefix_length: 8,
       scope: :host
     }
   ]},
  {["interface", "eth0", "state"], :configured},
  {["interface", "eth0", "hw_path"], "/devices/platform/scb/fd580000.ethernet"},
  {["interface", "eth0", "config"],
   %{ipv4: %{method: :dhcp}, type: VintageNetEthernet}},
  {["name_servers"],
   [
     %{address: {192, 168, 1, 149}, from: ["eth0"]},
     %{address: {1, 1, 1, 1}, from: ["eth0"]},
     %{address: {9, 9, 9, 9}, from: ["eth0"]}
   ]},
  {["interface", "lo", "present"], true},
  {["interface", "wlan0", "mac_address"], "e4:5f:01:72:ba:c3"}
]
```

**Comment by fhunleth** [comment]
This sounds good to add. What do you think about having one property that contains a map of every option returned by DHCP? I can imagine a few other fields that might eventually be convenient to have. Perhaps `["interface", ifname, "dhcp_options"]`?

**Comment by erauer** [comment]
I was considering  suggesting a single map property but there would be redundancy since properties such as `["interface", ifname, "addresses"]` are already being set elsewhere.

**Comment by fhunleth** [comment]
I'm not too worried about the redundancy. For example, the `"addresses"` property can contain more than DHCP-supplied IP addresses. If something turns out to be really obnoxious, we could remove it from the map property. I can't think of anything at the moment. In fact, your comment about `"addresses"` made me realize that if I ever had to debug an address issue, seeing what DHCP said without any interpretation would be a really nice piece of information.

**Comment by fhunleth** [comment]
Fixed in #442.
